client/app/scripts/superdesk/analytics/analytics.js

```diff
@@ -23,7 +23,9 @@ define([
 
         .run(['$rootScope', 'analytics', function($rootScope, analytics) {
             $rootScope.$on('$routeChangeSuccess', function(ev, route) {
-                analytics.track(route.$$route);
+                if (angular.isDefined(route)) {
+                    analytics.track(route.$$route);
```

Comment:
I'm not sure this is the best way, attributes prefixed by `$$` are considered internal to Angular and should not be directly. They might, for instance, change without notice in future framework versions.

Is there any other way to achieve the same functionality?